### ISSUE TITLE
Improve the quartz theme light mode

### ISSUE THREAD

**Issue by zoumingzhe** [adrianschubek/dir-browser#19]
**Describe the bug**
The information at the bottom of the quartz theme page is hide.

Similar issues also exist in other themes.

**Screenshots**
![image](https://github.com/adrianschubek/dir-browser/assets/23275497/c6f1be8c-a843-4b2f-ab41-0c9fce23f473)
![image](https://github.com/adrianschubek/dir-browser/assets/23275497/ea18f8bc-8215-4cad-9377-fc33bc37bf36)
![image](https://github.com/adrianschubek/dir-browser/assets/23275497/1820dd60-ad00-476a-b13e-5420334cea3e)
![image](https://github.com/adrianschubek/dir-browser/assets/23275497/e9021cc7-f188-4525-8608-af771b7fcd71)

**Comment by zoumingzhe** [comment]
No reverse display for light mode

![image](https://github.com/adrianschubek/dir-browser/assets/23275497/713ba164-fa4a-46cc-9510-18d06808d017)
![image](https://github.com/adrianschubek/dir-browser/assets/23275497/0f137c42-7fe6-4bd0-b7f1-aaa346714111)

**Comment by adrianschubek** [comment]
Yeah, I only made some optimizations for the default theme and most of the other themes are quite outdated.

**Comment by zoumingzhe** [comment]
Will we continue to support these themes?

**Comment by adrianschubek** [comment]
Some themes do still work but fixing them is low priority for now.
